installer/pkg/cmd/gcp_broker.go

```diff
@@ -464,8 +464,8 @@ func NewCreateGCPBrokerCmd() *cobra.Command {
 	cfg := &createBrokerConfig{}
 	cmd := &cobra.Command{
 		Use:   "create-gcp-broker",
-		Short: "Create a GCP broker",
-		Long:  "Creates a GCP broker without adding it to an existing cluster",
+		Short: "Create the Service Broker",
+		Long:  "Creates a Google Cloud Platform Service without adding it to an existing cluster",
```

Comment:
Google Cloud Platform Service Broker - "Broker" is missing here.

Reply:
Done